storage/pebble/config.go

```diff
@@ -14,7 +14,7 @@ func DefaultPebbleOptions(logger zerolog.Logger, cache *pebble.Cache, comparer *
 	opts := &pebble.Options{
 		Cache:              cache,
 		Comparer:           comparer,
-		FormatMajorVersion: pebble.FormatNewest,
+		FormatMajorVersion: pebble.FormatVirtualSSTables,
```

Comment:
Ensure after switching to pebble2, we can still switch back to pebble1 in case needed.